### PR TITLE
chore(#8259): Add module-alias to remove deep relative paths 

### PR DESCRIPTION
**Pull request by dianabarsan**
# Description

- adds module-alias dev dependency
- adds aliases for utils, factories, page-objects and constants
- aliases are used in all e2e suites
- moves some utils* files into /utils folder  

medic/cht-core#8259

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# Compose URLs

If Build CI hasn't passed, these may 404:

* [Core](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:module-alias/docker-compose/cht-core.yml)
* [CouchDB Single](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:module-alias/docker-compose/cht-couchdb.yml)
* [CouchDB Cluster](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:module-alias/docker-compose/cht-couchdb-clustered.yml)

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by dianabarsan** [comment]
Hey lovely reviewers! 

since this kinda saw the light of day in slack, I'm taking the opportunity to get some early feedback. 
It's a lot of work to keep this branch up to date - this is an indirect kudos for everyone pushing code to e2e tests, it's awesome!! - so I'll put in one big effort once this is approved. 

Thanks! Please don't get discouraged by the large # of changed files, because it's just repeated rewriting of the top includes section.

File: tests/e2e/protractor/contacts/reports-subject.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
-const utils = require('../../../utils');
-const commonElements = require('../../../page-objects/protractor/common/common.po.js');
-const reportsTab = require('../../../page-objects/protractor/reports/reports.po');
+const utils = require('@utils');
+const commonElements = require('@page-objects/protractor/common/common.po.js');
+const reportsTab = require('@page-objects/protractor/reports/reports.po');
 const helper = require('../../../helper');
```
Comment (lorerod) [inline tests/e2e/protractor/contacts/reports-subject.js:4]: It is there a reason not to use the alias with helper?

**Review by lorerod — APPROVED** [review]
Thank you, @dianabarsan, for this! It cleans the code well.

File: tests/e2e/default/navigation/run-guided-tour.wdio-spec.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
-const guidedTourPO = require('../../../page-objects/default/navigation/guided-tour.wdio.page.js');
-const loginPage = require('../../../page-objects/default/login/login.wdio.page');
-const commonPage = require('../../../page-objects/default/common/common.wdio.page');
+const guidedTourPO = require('@page-objects/default/navigation/guided-tour.wdio.page.js');
```
Comment (tatilepizs) [inline tests/e2e/default/navigation/run-guided-tour.wdio-spec.js:1]:
```suggestion
const guidedTourPage = require('@page-objects/default/navigation/guided-tour.wdio.page.js');
```

File: tests/e2e/default/sms/gateway.wdio-spec.js [hunk 3]
```diff
@@ -1,9 +1,9 @@
-const utils = require('../../../utils');
-const messagesPo = require('../../../page-objects/default/sms/messages.wdio.page');
-const reportsPo = require('../../../page-objects/default/reports/reports.wdio.page');
-const commonElements = require('../../../page-objects/default/common/common.wdio.page');
-const loginPage = require('../../../page-objects/default/login/login.wdio.page');
-const smsPregancy = require('../../../factories/cht/reports/sms-pregnancy');
+const utils = require('@utils');
+const messagesPo = require('@page-objects/default/sms/messages.wdio.page');
+const reportsPo = require('@page-objects/default/reports/reports.wdio.page');
```
Comment (tatilepizs) [inline tests/e2e/default/sms/gateway.wdio-spec.js:3]:
```suggestion
const messagesPage = require('@page-objects/default/sms/messages.wdio.page');
const reportsPage = require('@page-objects/default/reports/reports.wdio.page');
```

File: tests/e2e/default/translations/new-language.wdio-spec.js [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
-const commonPo = require('../../../page-objects/default/common/common.wdio.page');
-const utils = require('../../../utils');
-const sentinelUtils = require('../../../utils/sentinel');
-const userSettingsElements = require('../../../page-objects/default/users/user-settings.wdio.page');
-const contactsPage = require('../../../page-objects/default/contacts/contacts.wdio.page');
-const reportsPage = require('../../../page-objects/default/reports/reports.wdio.page');
-const loginPage = require('../../../page-objects/default/login/login.wdio.page');
-const messagesPage = require('../../../page-objects/default/sms/messages.wdio.page');
+const languagesPage = require('@page-objects/default/translations/languages.wdio.page');
+const commonPo = require('@page-objects/default/common/common.wdio.page');
```
Comment (tatilepizs) [inline tests/e2e/default/translations/new-language.wdio-spec.js:2]:
```suggestion
const commonPage = require('@page-objects/default/common/common.wdio.page');
```

**Review by tatilepizs — COMMENTED** [review]
This is great Diana, thank you very much for doing this change, it is clearer and easier to read, I love it 🤍 

I just have a favor to ask you, now that you are doing all of these changes, there are some files that have a name for the import as `reportsPo`, for example, but we are trying to standardize the name to `reportsPage`, for example. If you see some of those names can you please update them? (I will point the ones that I see on these files)

And about this that is in the description of the ticket:

> moves some utils* files into /utils folder

That is great, we have a ticket for it (#8224), it was on our list 😅 and probably would have needed help from you to do it properly, so thank you very much also for this change!!

**Comment by tatilepizs** [comment]
> Do you think our [automation test guide](https://github.com/medic/cht-core/blob/master/tests/AUTOMATE_TEST_GUIDE.md) will need to be updated to reflect this?

I am not sure @ngaruko, in the guide we talk about what will be the correct name to use in a new file, we are not talking about the path to use when a file is imported.

Here for example:

> Config file: [../tests/e2e/standard/wdio.conf.js](https://github.com/medic/cht-core/blob/master/tests/e2e/standard/wdio.conf.js)

we add the path so it is easier for a new person to find where is the config file located in the project.

And when we mention the name convention, like here: 

> Name convention for the spec file: ../tests/e2e/standard/*/<name>.wdio-spec.js

we are trying to explain where to locate a new file in the project, and what the name should look like.

Do you think that it might be confusing?

**Comment by dianabarsan** [comment]
@ngaruko I'm unsure if you have further feedback. Could you please have a look?

**Pull request merged**